### ISSUE TITLE
Controls still work when a probe loses signal

### ISSUE THREAD

**Issue by Fedex75** [JonnyOThan/RasterPropMonitor#124]
When a probe loses CommNet connection the monitors and controls still work.

[KSP.log](https://github.com/JonnyOThan/RasterPropMonitor/files/13687785/KSP.log)

1. Launch a probe
2. The probe loses signal
3. Go into probe control room (from the Probe Control Room Recontrolled plugin, but that shouldn't really matter)
4. Monitors still display information and controls still work

Expected behaviour: The monitors should turn off and controls shouldn't work, like when the probe runs out of electric charge.

![image](https://github.com/JonnyOThan/RasterPropMonitor/assets/48833722/0a87bbb4-93e1-4d09-a82c-b8fa392380c7)
Here we can see that the probe has no connection but the monitors are still working.

 - OS: Windows 10

**Comment by Fedex75** [comment]
Ok after looking at RasterPropMonitor.cs i think i understand it's more complex. So in order for the computer to disable the monitors when it loses signal, they would have to be configured with "requireCommConnection = true". The ALCORMFDs don't set that field so it's left false. But we can't change that directly on their cfg files because then all MFDs would require signal, even those inside command modules.

So the question then becomes, is there a way to change a field for an individual MFD?

**Comment by JonnyOThan** [comment]
Yeah good question...My hunch is that the solution here would be to have some variable on the RasterPropMonitorComputer (the partmodule that lives on the probe part) that controls the behavior.  Then all probes can be set up so that they require a signal and command pods don't.

**Comment by Fedex75** [comment]
Yeah that sounds good. On my local install i created new MFD props with their needsCommConnection always set to true and used those on the mission control cfg but that solution sucks.

By the way, this is unrelated but would it be possible to show the navigation marker on the MFD's navball?
I mean things like this:
![image](https://github.com/JonnyOThan/RasterPropMonitor/assets/48833722/cde37050-f988-4215-bef2-9c732a3a5035)
